### PR TITLE
feat(Tag): update to 2.0 styles

### PR DESCRIPTION
**Pull request by booc0mtaco**
- use new render color tokens
- update layout
- update tests and snapshots

### Test Plan:

- [x] Wrote/updated [automated tests](https://czi.atlassian.net/wiki/x/Hbl1H)
- [ ] CI tests / new tests are not applicable
- [ ] Manually tested my changes, and here are the details:

### CODE REVIEW

File: src/components/Tag/Tag.module.css [hunk 1]
```diff
@@ -45,6 +44,7 @@
 
 /**
  * Color variants.
+ * TODO(next-major): remove these status colors
```
Comment (ademartini-czi) [inline src/components/Tag/Tag.module.css:47]: Are these TODO tracked as tickets?

File: src/components/Tag/Tag.tsx [hunk 2]
```diff
@@ -1,72 +1,115 @@
...
-
 export type Variant = (typeof VARIANTS)[number];
 
 type Props = {
-  /**
-   * The color variant of the tag. It will update the content colors, background color, and border (when `hasOutline` is set to `true`).
-   *
-   * **Default is `"neutral"`**.
-   */
-  variant?: Variant;
```
Comment (ademartini-czi) [inline src/components/Tag/Tag.tsx:26]: Do we allow for breaking API contract with major version bump?

Comment (booc0mtaco) [reply src/components/Tag/Tag.tsx:26]: Yep, we do. Here, I moved `variant` to another part of `Props` for consistency (and to alphabetize better). those TODO(next-major) comments are for when we do the next major, likely in the new year. 

This one is backwards-compatible with some warnings (those `assertEDSUsage` log messages)

File: src/components/Tag/Tag.module.css (hunk 1 above)
Comment (booc0mtaco) [reply src/components/Tag/Tag.module.css:47]: Not currently, tho its not a bad idea to create one and add the detail of "delete all the deprecated things"

File: src/components/Tag/Tag.tsx (hunk 2 above)
Comment (ademartini-czi) [reply src/components/Tag/Tag.tsx:26]: OK got it, thanks

**Review by ademartini-czi — APPROVED** [review]

File: src/components/Tag/Tag.module.css (hunk 1 above)
Comment (booc0mtaco) [reply src/components/Tag/Tag.module.css:47]: Added a tracker to the backlog

**Pull request merged**